practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/configuration/PractitionerAttribute.java

```diff
@@ -0,0 +1,24 @@
+package uk.ac.ox.ndph.mts.practitioner_service.configuration;
+
+import lombok.Getter;
```

Comment:
Are we making a decision to use Lombok project wide? It makes sense to keep things consistent. There are some arguments for not using Lombok - annotation verbosity, especially when the class may be using Json data and JPA/Hibernate.. also potential issues with future versions of Java..although this is possible its unlikely..

Reply:
For simple POJOs Lombok is great. Not for JPA/Hibernate